runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/config/HttpAuthorizationConfigBuilder.java

```diff
@@ -26,7 +26,7 @@ public final class HttpAuthorizationConfigBuilder<T> extends ConfigBuilder<T, Ht
     private String name;
     private HttpCredentialsConfig credentials;
 
-    HttpAuthorizationConfigBuilder(
+    public HttpAuthorizationConfigBuilder(
```

Comment:
We don't expose these constructors elsewhere, instead we have a static method called `builder(...)` on the object being built, let's do that here too for consistency.